br/pkg/backup/schema.go

```diff
@@ -100,6 +110,11 @@ func (ss *Schemas) BackupSchemas(
 			schema.dbInfo.Name = utils.TemporaryDBName(schema.dbInfo.Name.O)
 		}
 
+		var checksum *checkpoint.ChecksumItem
```

Comment:
I think we should move this into `WorkerPool`

Reply:
we define it here to avoid the data race of map `Schemas::checkpointChecksum`.